src/webui/www/private/scripts/mocha-init.js

```diff
@@ -38,6 +38,13 @@
    ----------------------------------------------------------------- */
 "use strict";
 
+const limitWidthToViewport = (width) => {
+    return Math.min(width, window.innerWidth - 20);
+};
+const limitHeightToViewport = (height) => {
+    return Math.min(height, window.innerHeight - 75);
+};
+
```

Comment:
These functions are used by `window.qBittorrent.Dialog`, so they must be declared before the other functions in this file.

Reply:
Seems you can still move it into `window.qBittorrent.Dialog` (and export it) to avoid polluting the global.

Reply:
Good idea, moved to `window.qBittorrent.Dialog`